pkg/util/cgroups/pid_mapper.go

```diff
@@ -107,6 +107,19 @@ func getPidMapper(procPath, cgroupRoot, baseController string, filter ReaderFilt
 	return pidMapper
 }
 
+// getProcessNamespaceInode performs a stat() call on /proc/<pid>/ns/<namespace>
+// This has been copied from pkg/util/system/namespace_linux.go's GetProcessNamespaceInode
+func getProcessNamespaceInode(procPath string, pid string, namespace string) (uint64, error) {
```

Comment:
That should move to `namespace_linux.go`, the comment should state that the whole code should in `pkg/util/system/namespace_linux.go` but currently isn't due to isolation requirement made necessary by OpenTelemetry depending on the `trace-agent`.